playbook.tex

```diff
@@ -11,7 +11,7 @@
 
 \definecolor{AppearingCharacters}{RGB}{67, 67, 67}
 
-\newcommand{\character}[1]{\textbf{\MakeUppercase{#1}}}
+\newcommand{\character}[1]{\par\textbf{\MakeUppercase{#1}}}
```

Comment:
Just asking: what does the `\par` do here?

Reply:
Now the character command creates a new paragraph so it's not necessary to leave empty lines in the code, but you are still able to do it without creating two new paragraphs.